zk.go

```diff
@@ -5,13 +5,17 @@ import (
 	"path"
 	"sort"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/samuel/go-zookeeper/zk"
 )
 
 // ZK wraps a zookeeper connection
-type ZK struct{ *zk.Conn }
+type ZK struct {
+	*zk.Conn
+	chrootPath string
```

Comment:
Well, this is the term they use in the Kafka doc: http://kafka.apache.org/documentation.html